### PR TITLE
[dev] `binutils`: adding `libiberty` library and header

### PR DESCRIPTION
**Pull request by PawelWMS**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 

Adding the `libiberty` components to `binutils` to satisfy common requirements.

I've also updated `cairo` and `UI-cairo` builds. The new header and library triggers a build of the `symbol-lookup` feature, which fails to compile due to errors. Disabling that feature until required.

###### Change Log  

- Extended `binutils` with `libiberty`.
- Disabled `symbol-lookup` feature in `cairo` and `UI-cairo`.

###### Does this affect the toolchain?  

Yes.

###### Test Methodology

- Local package build.
- Pipeline toolchain build.

### CODE REVIEW

File: SPECS/cairo/cairo.spec [hunk 1]
```diff
@@ -69,7 +69,8 @@ It contains the libraries and header files to create applications
         --enable-xlib-xrender=no \
         --enable-gobject \
         CFLAGS="-O3 -fPIC" \
```
Comment (mfrw) [inline SPECS/cairo/cairo.spec:71]: This looks out of place to me .. although i understand this is not a change you introduced ..

File: SPECS/binutils/binutils.spec [hunk 2]
```diff
@@ -44,6 +46,9 @@ find %{buildroot} -type f -name "*.la" -delete -print
 rm -rf %{buildroot}%{_infodir}
 %find_lang %{name} --all-name
 
+install -m 644 libiberty/libiberty.a %{buildroot}%{_libdir}
+install -m 644 include/libiberty.h %{buildroot}%{_includedir}
```
Comment (mfrw) [inline SPECS/binutils/binutils.spec:50]: Double checking here; is `libiberty.h` the only header which we need to install or there is something else also ?

**Review by mfrw — APPROVED** [review]
LGTM :)

File: SPECS/binutils/binutils.spec (hunk 2 above)
Comment (PawelWMS) [reply SPECS/binutils/binutils.spec:50]: Just the header and the static lib.

File: SPECS/cairo/cairo.spec (hunk 1 above)
Comment (PawelWMS) [reply SPECS/cairo/cairo.spec:71]: True, I'm leaving it the way it is right now, to not destabilize the build too much.

**Pull request merged**